packages/core/src/rest/api/doc/IHeaderContent.ts

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Same question as above - curious about the "rest/api" folder

Reply:
fixed